controlpanel/frontend/views/app.py

```diff
@@ -74,6 +76,33 @@ def get_queryset(self):
         )
 
 
+class AppAdminCSV(OIDCLoginRequiredMixin, PermissionRequiredMixin, View):
+    permission_required = "api.is_superuser"
+
+    def post(self, request, *args, **kwargs):
+        apps = (
+            App.objects.filter(userapps__is_admin=True)
+            .annotate(
+                username=F("userapps__user__username"),
+                email=Coalesce("userapps__user__justice_email", "userapps__user__email"),
```

Comment:
I was wondering if it would be better to include both emails, but actually the justice email is probably the one you want to use in practice, as the other could be a non-work email as its based on whats linked to your GitHub. So actually think this is fine